src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    private static class Node<K, V> {
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_SIZE];
+        threshold = (int) (DEFAULT_SIZE * DEFAULT_LOAD_FACTOR);
```

Comment:
I think that you should take "table.length" not "DEFAULT_SIZE" during "threshold" calculation, because when table will increase more than one time your threshold would not be reliavant.